graphql-code-generator/src/main/java/io/ballerina/graphql/generator/service/generator/ServiceCodeGenerator.java

```diff
@@ -105,4 +182,4 @@ public void enableToUseRecords() {
     public void setServiceMethodDeclarations(List<MethodDeclarationNode> serviceMethodDeclarations) {
         this.serviceMethodDeclarations = serviceMethodDeclarations;
     }
-}
+}
```

Comment:
Add an empty line at the end of each file. You can configure your IDE to add this when saving a file.

```suggestion
}

```